lumicks/pylake/kymotracker/detail/localization_models.py

```diff
@@ -46,6 +46,21 @@ def __getitem__(self, item):
         return self.__class__(**{f.name: getattr(self, f.name)[item] for f in fields(self)})
 
 
+@dataclass(frozen=True)
+class CentroidLocalizationModel(LocalizationModel):
+    """Helper class to hold refinement optimization parameters.
+
+    Parameters
+    -----------
+    position : numpy.ndarray
+        Spatial coordinates in physical units.
+    total_photons : numpy.ndarray
+        Array of integrated photon counts for each time point.
+    """
+
+    total_photons: np.ndarray
```

Comment:
Why?

It would mean our default constructor would have to start taking photon counts (since it upconverts raw coordinates to a localization model) and we wouldn't be able to create just tracks without counts.

Right now we have several places where we generate `KymoTracks` when photon counts are unavailable, not interesting or don't really make sense.

I'd feel weird about including dummy counts when we don't really have them. I'd rather have the property be explicitly unavailable.

Reply:
ok, I see your point. although the times we generate tracks without actual photon counts is really just in the tests, so it still feels a bit weird to make API decisions just because of our testing infrastructure. But I guess it's ok leaving it like this